app/src/main/java/com/example/tanya/tatianaryabova/NewsItem.java

```diff
@@ -54,38 +54,40 @@ private NewsItem(@NonNull View itemView, RequestManager glideRequestManager, Con
     }
 
     public void bindItem(@NonNull NewsDTO news){
-        if(news.getMultimedia().size() != 0) {
-            imageLoader.load(news.getMultimediaUrl())
-                    .listener(new RequestListener<Drawable>() {
-                        @Override
-                        public boolean onLoadFailed(@Nullable GlideException e, Object model, Target<Drawable> target, boolean isFirstResource) {
-                            return false;
-                        }
+        if(news.getMultimedia() != null) {
```

Comment:
```suggestion
        if(news.getMultimedia() != null && news.getMultimedia().size() != 0) {
```
news.getMultimedia() тоже можно вынести в переменную